packages/evm/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@ethereumjs/evm",
-  "version": "1.1.0",
+  "version": "1.2.0",
```

Comment:
I guess I just noticed this but any reason we're using minor version updates for `evm` and point releases for the other packages?

Reply:
Yes. I already used a minor for the first release due to the significance of the event emitter update, for the VM as well, so to indicate that this would be a release with substantial changes (which was obviously still not enough 😋).

Reply:
Fair.  Lets merge it!

Reply:
Thanks for fixing this so quickly!

One note: for npm packages, package version numbers don't communicate significance (except version `1.0.0`) or provide any indication of how substantial the changes are. They indicate if the release contains fixes/refactors (0.0.x), new features (0.x.0), or breaking changes (x.0.0). Of course, if ethereumjs isn't adhering to [Semver](https://semver.org/) this isn't true, but we've been operating under the impression that it does.

Reply:
Hi there, yeah, Andrew did a really great job here! 🙂 👍 

I've just published all the versions, hope everything works now, otherwise let us know (respectively: some general feedback would likely be good in this case).

Yes, I know that (semver). We DO adhere to Semver. For the bugfix vs minor versions I perceive these lines as a bit blurry and not as important as for minor vs breaking. Feature additions are often so insignifcant for the user - if a new function is mainly for internal usage e.g., or if their is a slight API parameter change or so. If one would always call this out a minor version, one would very quickly land at v6.45. 😋

So I switched a bit (more) to a distinction to "what feels right to me" in these cases and I reserve minor versions more for the a bit more substantal feature additions.

Richard (from Ethers) is doing the opposite and is putting everything in bugfix releases. 😆